
References for the ESHG poster P26.014.C - presenter: Cappadona Claudio

Identification of genetic determinants of COVID-19 severity by exome sequencing

1- Barek A, Aziz A, Islam M S. Impact of age, sex, comorbidities and clinical symptoms on the severity of COVID-19 cases: A meta-analysis with 55 studies and 10014 cases. Heliyon, Volume 6, Issue 12, 2020. doi:10.1016/j.heliyon.2020.e05684

2- Severe Covid-19 GWAS Group. Genomewide Association Study of Severe Covid-19 with Respiratory Failure. N Engl J Med. 2020 Oct 15;383(16):1522-1534. doi: 10.1056/NEJMoa2020283. 

3- COVID-19 Host Genetics Initiative. Mapping the human genetic architecture of COVID-19. Nature 600, 472â€“477 (2021). https://doi.org/10.1038/s41586-021-03767-x

4- Van der Auwera GA & O'Connor BD. (2020). Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (1st Edition). O'Reilly Media.

5- GATK documentation: https://gatk.broadinstitute.org/hc/en-us/articles/360035535932-Germline-short-variant-discovery-SNPs-Indels- 

6- McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GR, Thormann A, Flicek P, Cunningham F. The Ensembl Variant Effect Predictor. Genome Biology Jun 6;17(1):122. (2016) doi:10.1186/s13059-016-0974-4

7- Ioannidis NM, Rothstein JH, Pejaver V, et al. REVEL: An Ensemble Method for Predicting the Pathogenicity of Rare Missense Variants. Am J Hum Genet. 2016;99(4):877-885. doi:10.1016/j.ajhg.2016.08.016

8- Guo MH, Plummer L, Chan Y-M, Hirschhorn JN, Lippincott MF. Burden testing of rare variants identified through exome sequencing using publicly available control data. American Journal of Human Genetics. 2018. 103(4):522-534.

9- Cirulli, E.T., White, S., Read, R.W. et al. Genome-wide rare variant analysis for thousands of phenotypes in over 70,000 exomes from two cohorts. Nat Commun 11, 542 (2020). https://doi.org/10.1038/s41467-020-14288-y

10- Lee S, Wu MC, Lin X. Optimal tests for rare variant effects in sequencing association studies. Biostatistics. 2012 Sep;13(4):762-75.

11- EPACTS. https://github.com/statgen/EPACTS

12- Zhou Y, Zhou B, Pache L, Chang M, Khodabakhshi AH, Tanaseichuk O, Benner C, Chanda SK. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nat Commun. 2019 Apr 3;10(1):1523.

13- Overmyer et al. Large-Scale Multi-omic Analysis of COVID-19 Severity. Cell Syst. 2021 Jan 20;12(1):23-40.e7. doi: 10.1016/j.cels.2020.10.003.

14- Lipman D, Safo SE, Chekouo T (2022) Multi-omic analysis reveals enriched pathways associated with COVID-19 and COVID-19 severity. PLoS ONE 17(4): e0267047.

15- Chen Q, Xia S, Sui H, Shi X, Huang B, Wang T (2022) Identification of hub genes associated with COVID-19 and idiopathic pulmonary fibrosis by integrated bioinformatics analysis. PLoS ONE 17(1): e0262737.

16- Leng L, Cao R, Ma J, et al. Pathological features of COVID-19-associated lung injury: a preliminary proteomics report based on clinical samples. Signal Transduct Target Ther. 2020;5(1):240. Published 2020 Oct 15. doi:10.1038/s41392-020-00355-9

17- Kosmicki JA et al; Regeneron Genetics Center; UKB Exome Sequencing Consortium. Pan-ancestry exome-wide association analyses of COVID-19 outcomes in 586,157 individuals. Am J Hum Genet. 2021 Jul 1;108(7):1350-1355.

18- Zhang S et al. Common and rare variant analyses combined with single-cell multiomics reveal cell-type-specific molecular mechanisms of COVID-19 severity. medRxiv [Preprint]. 2021 Jun 21:2021.06.15.21258703.

19- Kousathanas, A., Pairo-Castineira, E., Rawlik, K. et al. Whole genome sequencing reveals host factors underlying critical Covid-19. Nature (2022).

20- Zhang Q et al. Inborn errors of type I IFN immunity in patients with life-threatening COVID-19. Science. 2020 Oct 23;370(6515):eabd4570.

21- D'Alterio G et al. Germline rare variants of lectin pathway genes predispose to asymptomatic SARS-CoV-2 infection in elderly individuals. Genet Med. 2022 May 5:S1098-3600(22)00724-9.
